python/get_python.sh

```diff
@@ -32,7 +32,8 @@ install_dependencies () {
         pushd $DIR/../scripts/o3de/
         $DIR/python.sh setup.py sdist
         popd
-
+    fi
+    if [ -f $DIR/../scripts/o3de/dist/o3de-1.0.0.tar.gz ]; then
```

Comment:
Can you add a comment clarifying this is checking the o3de python scripts setup version of 1.0.0 which is set in [scripts/o3de/setup.py](https://github.com/o3de/o3de/blob/development/scripts/o3de/setup.py#L28)?